source/Octopus.Client.Tests/Operations/RegisterMachineOperationFixture.cs

```diff
@@ -47,28 +47,17 @@ public void SetUp()
                     .Add("CurrentUser", "/api/users/me")
                     .Add("SpaceHome", "/api/spaces")
             };
-            client.Get<RootResource>(Arg.Any<string>()).Returns(rootDocument);
-            client.Repository.LoadRootDocument().Returns(rootDocument);
-            client.Get<SpaceResource[]>(Arg.Any<string>())
-                .Returns(new[] {new SpaceResource() {Id = "Spaces-1", IsDefault = true}});
-            client.Get<UserResource>(Arg.Any<string>()).Returns(new UserResource()
-            {
-                Links =
-                {
-                    {"Spaces", ""}
-                }
-            });
```

Comment:
Are these protecting us from anything, or just unnecessary waffle?

Reply:
The path is not accessed at all. Probably a copy paste from other test that uses it.